docs/_sass/table.scss

```diff
@@ -20,7 +20,7 @@ table th {
 .td-box .row.section > table td,
 .td-content > table td {
   padding: 1rem;
-  vertical-align: top;
```

Comment:
Did you confirm that these changes don't negatively affect other table uses, like the integration test results?